### DISCUSSION TITLE
Welcome to Precedent Discussions!

### DISCUSSION THREAD

**Discussion by steven-tey** [steven-tey/precedent#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by schonert** [comment]
Well done! A beautifully, polished package of goodies triggering ever bone of mine to start a side project 💪🏻

And the logo is brilliant, nothing less 🤌🏻

**Comment by 2ne** [comment]
When you use the Precedent template via Vercel there does not seem to be an easy way to port fixes that you have done to the repo. Any ideas how to fix?

**Comment by akashp1712** [comment]
I'm using the template for my micro-saas at https://lessentext.com. Can I extend it for dashboard and app module ?

**Comment by picklebrownie** [comment]
Thank you for sharing this 💯 👏🏼
